nvflare/private/fed/server/server_json_config.py

```diff
@@ -163,7 +161,7 @@ def finalize_config(self, config_ctx: ConfigContext):
         )
 
         ConfigService.initialize(
-            section_files={SystemConfigs.APPLICATION_CONF: os.path.basename(self.config_files[0])},
+            section_files={},
```

Comment:
Do we need this "section_files" arguments in "ConfigService" anymore?
Should we remove it and related get/set sections in follow PRs?

Reply:
The idea of the "section_files" is to load commonly needed configs (e.g. server config - not job config) once and let all other places use them. But this idea wasn't used properly. We need to have a consistent config approach. Right now there are different ways. Let's discuss these later. For now let's keep section files.